time-to-live.md

```diff
@@ -183,4 +238,32 @@ Currently, the TTL feature has the following limitations:
+
+   ! [insert fast example](/media/ttl/insert-fast.png)
+
+   It is worth noting that since TTL does not guarantee that the expired rows will be deleted immediately, and the rows currently inserted will be deleted in a future TTL task, even if the speed of TTL deletion is lower than the speed of insertion in a short period of time, it does not necessarily mean that the speed of TTL is too slow. It needs to be considered in the context.
+
+- How can I determine whether the bottleneck of a TTL task is in scanning or deleting?
+
+   Look at the `TTL Scan Worker Time By Phase` and `TTL Delete Worker Time By Phase` panels. If the scan worker is in the `dispatch` phase for a large percentage of time and the delete worker is rarely in the `idle` phase, then the scan worker is waiting for the delete worker to finish the deletion work, and if the cluster resources are still free at this point, you can consider increasing the `tidb_ttl_ delete_worker_count` to increase the number of delete workers. For example.
+
+   ! [scan fast example](/media/ttl/scan-fast.png)
+
+   In contrast, if the scan worker is rarely in the `dispatch` phase and the delete worker is in the `idle` phase for a long time, then the scan worker is relatively busy. For example.
+
+   ! [delete fast example](/media/ttl/delete-fast.png)
+
+   The percentage of scan and delete in TTL jobs is related to the machine configuration and data distribution, so the monitoring data at each moment is only representative of the TTL Jobs being executed. Users can read the table `mysql.tidb_ttl_job_history` to determine which TTL job is running at a certain moment, and the corresponding table of the job.
+
+- How to configure `tidb_ttl_scan_worker_count` and `tidb_ttl_delete_worker_count` properly?
+
+   First, you can refer to the question "How to determine whether the bottleneck of TTL tasks is in scanning or deleting?" to consider whether to boost `tidb_ttl_scan_worker_count` or `tidb_ttl_delete_worker_count`. Also, if the number of TiKV nodes is high, boosting `tidb_ttl_scan_worker_count` can make the TTL task load more even.
+
+   Since too many TTL workers will cause a lot of stress, you need to look at the CPU level of TiDB and the disk and CPU usage of TiKV together. Depending on different scenarios and needs (need to speed up TTL as much as possible, or need to reduce the impact of TTL on other queries), `tidb_ttl_scan_worker_count` and `tidb_ttl_delete_worker_count` can be increased to improve the speed of TTL scanning and deleting data, or reduce the performance impact brought by TTL tasks.
```

Comment:
```suggestion
   Since too many TTL workers will cause a lot of pressure, you need to evaluate the CPU level of TiDB and the disk and CPU usage of TiKV together. Depending on different scenarios and needs (whether you need to speed up TTL as much as possible, or to reduce the impact of TTL on other queries), you can adjust the value of `tidb_ttl_scan_worker_count` and `tidb_ttl_delete_worker_count` to improve the speed of TTL scanning and deleting or reduce the performance impact brought by TTL tasks.
```